File: src/main/java/com/back/together02be/chart/dto/Candle.java
package com.back.together02be.chart.dto;

import java.math.BigDecimal;
import java.time.LocalDate;

public record Candle(
	LocalDate date,
	BigDecimal open,
	BigDecimal high,
	BigDecimal low,
	BigDecimal close,
	Long volume
) {
}

File: src/main/java/com/back/together02be/chart/dto/Candle.kt
package com.back.together02be.chart.dto

import java.math.BigDecimal
import java.time.LocalDate

@JvmRecord // todo @JvmRecord 제거
data class Candle(
	val date: LocalDate,
	val open: BigDecimal,
	val high: BigDecimal,
	val low: BigDecimal,
	val close: BigDecimal,
	val volume: Long
)
